support-workers/scripts/update-code-cloudformation.sh

```diff
@@ -1,24 +1,17 @@
 #!/bin/bash
 
-./build-cloudformation.sh
```

Comment:
do we need to run pnpm build or similar at this point?

Reply:
Yes, it's running `yarn synth`

Reply:
I must be blind it looks like
`./build-cloudformation.sh`
has been replaced with  a comment
`# We need to upload the template to S3 because it is too big to inline`

Reply:
Realised I didn't push this change!